jenkins/custom-ci/centos-7.9.2009/motr.groovy

```diff
@@ -67,7 +67,7 @@ pipeline {
 						rm -rf /root/rpmbuild/RPMS/x86_64/*.rpm
 						KERNEL=/lib/modules/$(yum list installed kernel | tail -n1 | awk '{ print $2 }').x86_64/build
```

Comment:
Please remove this line also.

Reply:
but In actual file I can't see this line  https://github.com/nikhilpatil2995/cortx-re-1/blob/EOS-26885-motr-user-mode-rpms/jenkins/custom-ci/centos-7.9.2009/motr.groovy

Reply:
okay